recipes/pamela/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: pamela
+  version: {{ version }}
+
+source:
+  fn: pamela-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/43/d1/21403bbafd8729d441ba45e145810f4b7a1f339718827de336cc7246c450/pamela-{{ version }}.tar.gz
```

Comment:
We are preferring the pypi.io URLs instead of the new (ugly) ones:

`https://pypi.io/packages/source/p/pamela/pamela-{{ version }}.tar.gz`